exasol_script_languages_container_ci/lib/ci_test.py

```diff
@@ -4,22 +4,29 @@
 import click
 from exasol_script_languages_container_tool.cli.commands import run_db_test
 
-import exasol_script_languages_container_ci
+from exasol_script_languages_container_ci.lib.branch_config import BranchConfig
 from exasol_script_languages_container_ci.lib.common import print_docker_images
 from exasol_script_languages_container_ci.lib.git_access import GitAccess
 
 
-def ci_test(ctx: click.Context, check_for_skip_test: bool, flavor_path: Tuple[str, ...], git_access: GitAccess):
+def _need_to_run_tests(branch_name: str, git_access: GitAccess):
```

Comment:
You might need to want to test this function, not sure making it private is then a good idea

Reply:
I thought about adding an explicit test, but the function is so trivial. I have the `test_test` module which tests several aspects of the test logic, also the branches and the commit messages. So actually this part is covered by the test